### ISSUE TITLE
Add download modes: csv, parquet, json

### ISSUE THREAD

**Issue by jesi-rgb** [jesi-rgb/tractor2.0#1]
As per Jorge's suggestion, we can also rename the columns to be {columnName}<gx:{columnType}>, which skips the type inference altogether, making projects load particularly faster.

Example: numViews is of type number -> rename to numViews<gx:number>

Graphext will remove `<gx:number>` internally.

**Comment by jorses** [comment]
Yep, you can check out the actual code [here](https://github.com/graphext/grapy/blob/768ab6108cc69dec16425de812ae6364fe79e055/libs/core/grapy/core/wrangle/sanitize.py).

Semantic names used usually are the shortform lowercase kind e.g. list[category] or list[number], number, category, url ...
